i18n/pt-br.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  name: 'Português (Brasil)',
+  key: 'pt-br',
```

Comment:
Maybe it's better to use 'pt-BR' key to exactly match osrm-text-instructions' language pack name?